modules/machineset-yaml-azure.adoc

```diff
@@ -105,14 +105,17 @@ endif::infra[]
           resourceGroup: <infrastructure_id>-rg <1>
           sshPrivateKey: ""
           sshPublicKey: ""
+          tags:
+            - name: <custom_tag_name> <9>
```

Comment:
Just checking that the `-` character before the `name` field is supposed to be in there (I didn't see it in the bug text)

Reply:
nice observation! yes, it is required.